.github/workflows/run-frequent.yaml

```
@@ -123,3 +124,4 @@ jobs:
       issue_num: ${{ inputs.issue_num }}
       issue_hash_prefix: Testsuite Status
       prefix: ''
+      branch: 'master'
```

```suggestion
      branch: master
```
or the other way - just match the style of the other callsite